### PR TITLE
Feature/weos 939

### PR DESCRIPTION
**Pull request by EmmanuelSammy**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- updates to update projections to overwrite or point to newly saved data.

### CODE REVIEW

File: src/gorm_projection.go [hunk 1]
```diff
@@ -143,7 +150,7 @@ var AddQuery QueryModifier
 func (c *GORMProjection) GetCustomers(page int, limit int, query string, sortOptions *[]string, filterOptions map[string]interface{}) ([]*Customer, int64, error) {
 	var customers []*Customer
 	var count int64
-	result := c.db.Preload("Addresses").Preload("Emails").Preload("Phones").Scopes(filter(filterOptions), paginate(page, limit), sort(sortOptions, "customers"), AddQuery(query)).Find(&customers).Offset(-1).Distinct("customers.ID").Count(&count)
+	result := c.db.Debug().Preload("Addresses").Preload("Emails").Preload("Phones").Scopes(filter(filterOptions), paginate(page, limit), sort(sortOptions, "customers"), AddQuery(query)).Find(&customers).Offset(-1).Distinct("customers.ID").Count(&count)
```
Comment (atoniaw) [inline src/gorm_projection.go:153]: do we still need the debug arguement?

File: src/projections.go [hunk 2]
```diff
@@ -145,12 +145,8 @@ type Tag struct {
 
 type Projection interface {
 	weos.Projection
-<<<<<<< HEAD
-	GetCustomers(page int, limit int, query string, order *[]string, filter *map[string]interface{}) ([]*Customer, int64, error)
-	GetCustomerByID(id string) (*Customer, error)
-=======
 	GetCustomers(page int, limit int, query string, order *[]string, filter map[string]interface{}) ([]*Customer, int64, error)
->>>>>>> origin/dev
```
Comment (atoniaw) [inline src/projections.go:153]: is this properly merged?

File: src/gorm_projection.go (hunk 1 above)
Comment (EmmanuelSammy) [reply src/gorm_projection.go:153]: not sure tbh it works fine without it

File: api.yaml [hunk 3]
```diff
@@ -114,13 +114,6 @@ paths:
             type: string
           required: true
           description: id of the customer
```
Comment (atoniaw) [inline api.yaml:116]: add authenticate middleware this route

**Review by atoniaw — APPROVED** [review]

**Pull request merged**